### DISCUSSION TITLE
Hackathon planning

### DISCUSSION THREAD

**Discussion by bmann** [DWebYVR/notes#30] [Planning & Suggestions]
Let's do a hackathon! Including defining what that word means :)

Ideally we can try out The Permanent for this kind of event, and have people "hacking" on whatever they like.

Maybe: some initial demos / lightning talks about certain tech or APIs?

Either on a weekend day or an afternoon until late (maybe 4pm - 8pm) weekday evening, so that those who work outside of their day jobs can join.

Aim for a modest amount of attendees -- e.g. no more than 50 to start.

---

What: One day hackathon / code together

Attendees: Max 50 -- we'll be happy with 25

Maybe: some initial demos / lightning talks

Timing: Sat July 22nd or 29th

* [ ] Check on The Permanent availability
* [ ] Other venue research
* [ ] Power bars

**Comment by bmann** [comment]
@weswalla suggests July. If we give ourselves 2 months from now, here are some potential dates:

Weekday evenings:
* Wednesday, July 19th
* Thursday, July 20th
* Tuesday, July 25th
* Thursday, July 27th

Weekend day:
* Saturday July 22nd
* Saturday, July 29th

**Reply by rithikha** [reply]
Not sure if I can make it but might be able to do a weekend - either Saturday July 22 or 29th!

**Comment by bmann** [comment]
OK we'll focus on a weekend day!

**Comment by bmann** [comment]
Talked about this at the garden gathering https://github.com/orgs/DWebYVR/discussions/34 -- summer in Vancouver should be spent outside!

Kick off the fall indoor time with a September hackday on one weekend. Going to close this thread and start a new one.
